service/network/network.go

```diff
@@ -0,0 +1,4 @@
+// Package network is DEPRECATED and can be dropped once v29 is gone. The
```

Comment:
Why not https://github.com/giantswarm/fmt/blob/master/go/todo_comments.md ?

Reply:
You are right. Done in https://github.com/giantswarm/aws-operator/pull/1745.